Automattic/simplenote-android
# Update Widget Preview

### Fix
Update the note widget preview images.  The corner radius of widgets was changed from 2dp to 4dp to follow the [Material guidelines for shaping](https://material.io/design/shape/about-shape.html#shaping-material).  See the screenshots below for illustration.

![note_widget_preview](https://user-images.githubusercontent.com/3827611/74686491-e6f14980-518e-11ea-9561-78e3b82dc18b.png)

### Test
These steps assume the AOSP launcher (i.e. Nexus/Pixel launcher). The steps to add the widget to the home screen may be slightly different based on the launcher used. Other steps should be the same regardless of launcher.
1. Long-press home screen.
2. Tap ***Widgets*** option.
3. Scroll to ***Simplenote*** widget.
4. Notice ***Note (Dark)*** and ***Note (Light)*** widgets with updated preview images.

### Review
Only one developer and one designer are required to review these changes, but anyone can perform the review.

### Release
These changes do not require release notes.
